### PR TITLE
Reformat docs and menu

### PR DESCRIPTION
**Pull request by jclark-vmware**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

https://jira.carbonblack.local/browse/CBAPI-4739

## Pull Request Description

reformatting docs

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/getting-started.rst [hunk 1]
```diff
@@ -152,3 +152,9 @@ how you could identify yourself:
   integration_name=MyScript/0.9.0
 
 See the :doc:`authentication` documentation for more information about credentials.
+
+Next Steps
+----------
+
+ - :doc:`concepts`: General information about retrieving data from your Carbon Black Cloud instance
```
Comment (avanbrunt-cb) [inline docs/getting-started.rst:159]: I would rephrase concepts more towards the nuance of using the CBC SDK as it describes the python constructs more than the Carbon Black Cloud data

**Review by avanbrunt-cb — APPROVED** [review]
Are you going to create a separate PR for breaking up the package files into smaller files?

**Comment by jclark-vmware** [comment]
> Are you going to create a separate PR for breaking up the package files into smaller files?

Yes, that is my plan

**Review by abowersox-cb — APPROVED** [review]
